formatTest/typeCheckedTests/expected_output/fastPipe.re

```diff
@@ -0,0 +1,119 @@
+/* tests adapted from https://github.com/BuckleScript/bucklescript/blob/master/jscomp/test/pipe_syntax.ml */
+include (
+          {
+            let t0 = (x, f) => x ->f ->f ->f;
+            let t1 = (x, f) => x ->f;
+            let t2 = (x, f, g) =>
+              x ->f ->g(x, x) ->f(x);
+            let t3 = (x, f) =>
+              x ->f(~h=1, ~x=2);
+          }: {
+            let t0: ('a, 'a => 'a) => 'a;
+            let t1: ('a, 'a => 'b) => 'b;
+            let t2:
+              (
+                'a,
+                ('a, 'a) => 'b,
+                ('a => 'b, 'a, 'a) => 'a
+              ) =>
+              'b;
+            let t3:
+              (
+                'a,
+                ('a, ~h: int, ~x: int) => 'b
+              ) =>
+              'b;
+          }
+        );
+
+let f = (a, b, c) => a ->(b, c);
```

Comment:
Can we remove the space between `a` and `->`?

Reply:
Yes, on my todolist, still thinking this through, there are some challenges 🐒